tools/mOTUs/motus_merge.xml

```diff
@@ -0,0 +1,90 @@
+<tool id="mereg_mOTUs_tables" name="Merge" version="0.1.0+galaxy0" profile="21.05">
+    <description>Merge mOTUs aboundance or count tables</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="biotools"/>
+    <expand macro="requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+    #if $db.db_source == 'cached'
+            #set $DATABASE_DIR=$db.db_cached.fields.path
+        #else
+            #set $database_path = $db.database
+            file $database_path &&
+            mkdir database_dir &&
+            tar -xvf '$database_path' -C database_dir/ &&
+            #set $DATABASE_DIR = 'database_dir/db_mOTU' ##the actual DB folder in the archive called db_mOTU
+        #end if
+        echo "Database Directory:" $DATABASE_DIR  &&
+        motus merge
+        -db '$DATABASE_DIR'
+        -i '${','.join([str($file) for $file in $files])}'
+        #if $add_profiles
+        -a '$add_profiles'
+        #end if
+        -o '$output'
+    $biom_format
+    ]]></command>
+    <inputs>
+        <param name="files" type="data" format="txt" multiple="true" label="mOTU profiles to merge" />
+        <param name="add_profiles" type="text" label="Add pre-computed profiles" help="Add pre-computed profiles from different environmental samples" optional="true"/>
+        <param name="biom_format" type="boolean" truevalue="-B" falsevalue="" checked="false" label="Output in BIOM format?" />
+
+        <conditional name="db">
+            <param name="db_source" type="select" label="mOTUs Database">
+                <option value="cached">Use a pre-installed database</option>
+                <option value="history">Use a database from history</option>
+            </param>
+            <when value="cached">
+                <param name="db_cached" type="select" label="A pre-installed mOTUs databese" help="no pre-installed mOTUs databases are avalable yet">
+                    <options from_data_table="mOTUs_db">
+                        <column name="value" index="0"/>
+                        <column name="name" index="1"/>
+                        <column name="version" index="2"/>
+                        <column name="path" index="3" />
+                        <filter type="sort_by" column="1"/>
+                    </options>
+                </param>
+            </when>
+            <when value="history">
+                <param name="database" type="data" format="tar" label="Database from history" help="Custom database or other version of mOTUs database from link: https://zenodo.org/records/5140350"/>
+            </when>
+        </conditional>
+          
+    </inputs>
+    <outputs>
+        <data name="output" format="txt" from_work_dir="." label="${tool.name} on ${on_string}: Merged taxon relative abundance tables"/>
```

Comment:
what if a user changes the datatype to BIOM, this is still a text, please change the datatype?

Reply:
https://docs.galaxyproject.org/en/latest/dev/schema.html#id85